src/components/entity/ha-state-label-badge.html

```diff
@@ -14,6 +14,15 @@
 
       ha-label-badge {
         --ha-label-badge-color: var(--label-badge-red, #DF4C1E);
+        --ha-label-badge-label-text-transform: uppercase;
```

Comment:
No need to set it since it should fallback to using uppercase if var not defined.